draft-ietf-httpbis-client-hints.md

```diff
@@ -162,55 +163,59 @@ The opt-in MUST be ignored if delivered over non-secure transport (using a schem
+It will then use it for navigations to e.g., "https://site.example/foobar.html", but not to e.g., "https://foobar.site.example/".
+It will similarly use the preference for any same-origin resource requests (e.g., to "https://site.example/image.jpg") initiated by the page constructed from the navigation's response, but not to cross-origin resource requests (e.g., "https://thirdparty.example/resource.js").
+This preference will not extend to resource requests initiated to "https://site.example" from other origins (e.g., from navigations to "https://other.example/").
 
 ## Interaction with Caches
 
-When selecting a response based on one or more Client Hints, and if the resource is cacheable, the server needs to generate a Vary response header field ({{RFC7234}}) to indicate which hints can affect the selected response and whether the selected response is appropriate for a later request.
+When selecting a response based on one or more Client Hints, and if the resource is cacheable, the server SHOULD generate a Vary response header field ({{RFC7234}}) to indicate which hints can affect the selected response and whether the selected response is appropriate for a later request.
 
 ~~~ example
   Vary: Sec-CH-Example
 ~~~
 
-Above example indicates that the cache key needs to include the Sec-CH-Example header field.
+The above example indicates that the cache key needs to include the Sec-CH-Example header field.
 
 ~~~ example
   Vary: Sec-CH-Example, Sec-CH-Example-2
 ~~~
 
-Above example indicates that the cache key needs to include the Sec-CH-Example and Sec-CH-Example-2 header fields.
+The above example indicates that the cache key needs to include the Sec-CH-Example and Sec-CH-Example-2 header fields.
 
 
 # Security Considerations
 
 ## Information Exposure
-Request header fields used in features relying on this document expose information about the user's environment to enable proactive content negotiation. Such information might reveal new information about the user and implementers ought to consider the following considerations, recommendations, and best practices.
 
-The underlying assumption is that exposing information about the user as a request header is equivalent (from a security perspective) to exposing this information by other means. (For example, if the request's origin can access that information using JavsScript APIs, and transmit it to its servers).
+Request header fields used in features relying on this document expose information about the user's environment to enable privacy-preserving proactive content negotiation, and avoid exposing passive fingerprinting vectors.
+However, implementers need to bear in mind that in the worst case, uncontrolled and unmonitored active fingerprinting is not better than passive fingerprinting. In order to provide user privacy benefits, user agents need to apply further policies that prevent abuse of the information exposed by features using Client Hints. 
```

Comment:
Thanks, this is a good way to frame the subsequent discussion to keep the relevant topics prominent.

Reply:
Thanks! :)